site/styles/_pages.scss

```diff
@@ -70,8 +72,6 @@
   }
 }
 .form-page {
-  margin: 0 auto;
-  max-width: 1600px;
```

Comment:
We don't want this constraint anymore?

Reply:
The margin: 0 auto was was squashing the width of the forms once I changed the page to flexbox. Once the max-width reached 1600px, the .form-page container would not extend across the width of the page. This resulted in the form being off centre. 

![image](https://user-images.githubusercontent.com/41402426/95149032-634da100-07e1-11eb-9bab-79fdf2e0c4a3.png)

Reply:
Cool